.github/workflows/build-sphinx.yml

```diff
@@ -91,7 +91,7 @@ jobs:
 
       - name: Install dpnp dependencies
         run: |
-          conda install numpy"<1.24" dpctl mkl-devel-dpcpp onedpl-devel tbb-devel dpcpp_linux-64 \
+          conda install numpy"<1.24" dpctl">=0.15.0" mkl-devel-dpcpp onedpl-devel tbb-devel dpcpp_linux-64 \
```

Comment:
```suggestion
          conda install numpy"<1.24" dpctl">=0.15.1*" mkl-devel-dpcpp onedpl-devel tbb-devel dpcpp_linux-64 \
```

Reply:
The change in `dpnp` requires `dpctl` where public API to get internal implementation routine has been added. The API is not available in 0.15.0